target/linux/ar71xx/image/tp-link.mk

```diff
@@ -384,6 +384,18 @@ define Device/tl-mr13u-v1
 endef
 TARGET_DEVICES += tl-mr10u-v1 tl-mr11u-v1 tl-mr11u-v2 tl-mr12u-v1 tl-mr13u-v1
 
+define Device/tl-mr22u-v1
+  $(Device/tplink-8mlzma)
+  DEVICE_TITLE := TP-LINK TL-MR22U V1
+  DEVICE_PACKAGES := kmod-usb-core kmod-usb2 kmod-usb-ledtrig-usbport
+  BOARDNAME := TL-MR22U
+  DEVICE_PROFILE := TLMR22U
+  TPLINK_HWID := 0x00220101
+  CONSOLE := ttyATH0,115200
```

Comment:
For QCA953x based devices this value is wrong. It should be `ttyS0,115200` and as [it's the default value](https://github.com/lede-project/source/blob/master/target/linux/ar71xx/image/Makefile#L105), you can just drop whole line.